hack/start-verify-ec-task.sh

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/bash
+#
+# Rough hacking guide:
+#
+#  * Commit your changes as required. (No need to push to GitHub.)
+#
+#  * Push a new base image and new task bundles like this:
+#      env MY_QUAY_USER=$USER BUILD_TAG=$(git rev-parse HEAD) ./build-and-push.sh
+#
+#    (Assumes you have the required repos created in quay.io, you're signed in there with podman,
+#    and your quay.io username matches $USER)
+#
+#  * Create a Secret in the current namespace containing the cosign.pub key. This should be a
+#    partial copy of the Secret tekton-chains/signing-secrets. Only the public key should be
+#    included in this Secret. By default the name of the Secret is 'cosign-public-key', set
+#    the PUBLIC_KEY_SECRET environment variable to use a different name.
+#
+#  * Make sure you have an image that has already been signed by Chains.
+#
+#  * (Optional) Modify the POLICY_REPO and POLICY_REPO_REF params below as required.
+#
+#  * Run this script:
+#      ./start-vec-task.sh <image-ref>
```

Comment:
Seems like a previous version of the script name here.

Reply:
Thanks for catching it! Fixed.